### PR TITLE
[CD1] bugfix: duplicate networks when adding to the addToFrequentRPClist

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Description**

This PR fixes https://github.com/MetaMask/mobile-planning/issues/307#issuecomment-1160859054

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Screenshots/Recordings**

_If applicable, add screenshots or recordings to visualize the changes_

**Issue**

Progresses #???

### CODE REVIEW

File: app/components/Views/Settings/NetworksSettings/NetworkSettings/index.js [hunk 1]
```diff
@@ -449,7 +449,7 @@ class NetworkSettings extends PureComponent {
       // Remove trailing slashes
       const formattedHref = url.href.replace(/\/+$/, '');
       PreferencesController.addToFrequentRpcList(
-        formattedHref,
+        url.href,
```
Comment (sethkfman) [inline app/components/Views/Settings/NetworksSettings/NetworkSettings/index.js:452]: Is there a reason why the original URL is fixing this issue?

Comment (blackdevelopa) [reply app/components/Views/Settings/NetworksSettings/NetworkSettings/index.js:452]: Yeah. The idea is to save the rpcUrl the exact way without any formatting so the controllers doesn't assume it's a different url and create a new copy.

**Review by sethkfman — APPROVED** [review]
LGTM

**Review by cortisiko — APPROVED** [review]
This is QA passed 🌮 🌮

**Pull request merged**